cmd/root.go

```diff
@@ -141,6 +148,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.openshift-goimports.yaml)")
 
 	rootCmd.Flags().StringVarP(&path, "path", "p", "", "The path to the go module to organize. Defaults to the current directory.")
+	rootCmd.Flags().StringVarP(&intermediates, "intermediates", "i", "", "Space-separated list of names of go modules to put between openshift and module to organize. Example: github.com/thirdy/one thirdy.io/two")
```

Comment:
Can you update this to use `StringSlice` which will accept a comma separated list of values, or `StringArray` which accepts multiple invocations of the flag, both of which negate the need to later split the list manually.

Reply:
Done in latest rev.